src/content/zh-CN/2020/performance.md

```diff
@@ -0,0 +1,396 @@
+
+## 首字节时间
+
+首字节时间（TTFB）指的是初始的 HTML 请求到浏览器接收到第一个字节的时间。能否快速处理请求的问题会迅速影响其他性能指标，因为它们不仅会延迟绘制，还会延迟任何资源的获取。
+
+### TTFB 设备分类
+
+{{ figure_markup(
+  image="performance-ttfb-by-device.png",
+  caption="按设备类型分类的 TTFB 总性能",
+  description="条形图显示，有76％的网站在桌面端上的 TTFB 较差，在移动设备上这一比例为83％。具有良好TTFB的网站数量不超过24％。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=1981576071&format=interactive",
+  sheets_gid="1270303192",
+  sql_file="web_vitals_by_device.sql"
+  )
+}}
+
+在桌面端，76%的站点有着不太好的 TTFB，然而在移动端，这个比例上升到了83%。我们可以假设这个数据描述了 TTFB 是一个通常被忽略的指标，大多数性能测量和工作都集中在了前端和视觉渲染，而不是资源伺服和服务端工作。高 TTF 将对大量其他性能指标产生直接的负面影响，而这仍然是需要解决的问题。
+
+### TTFB 地理位置分布
+
+{{ figure_markup(
+  image="performance-ttfb-by-geo.png",
+  caption="按国家分类的 TTFB 总性能",
+  description="条形图显示 TTFB 的性能始终低于标准，在28个国家中，只有4个拥有超过30％的 TTFB 良好的网站。有大量的网站被归类为需要改善（始终高于40％），不良体验的比例随着排名的降低而上升。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=1135415956&format=interactive",
+  width="645",
+  height="792",
+  sheets_gid="1440255644",
+  sql_file="web_vitals_by_country.sql"
+  )
+}}
+
+今年的 TTFB 地理分布图和[2019年结果](../2019/performance#ttfb-by-geo)相似，都表明了有更多的站点变快了，但是和 FCP 相似，阈值都发生了改变。以前，我们认为 TTFB 在200ms以下为快，而在1000ms以上为慢。 2020年，低于500ms的TTFB良好，高于1500ms的TTFB较差。这些评级方面的许多变化可以解释我们观察到的重大改变，比如韩国有着良好用户体验的站点增加了36％，台湾增加了22%。总体上，我们仍然看到了一些和往年的相似点，例如亚太地区和某些欧洲地区仍然处于领先地位。
+
+### 按连接类型查看 TTFB
+
+{{ figure_markup(
+  image="performance-ttfb-by-connection-type.png",
+  caption="按连接类型分类的 TTFB 总性能", 
+  description="条形图显示，TTFB 受到连接类型的严重影响，在4G和离线的场景下，分别只有21％和22％的体验良好。其他连接类型几乎没有良好的 TTFB 体验（3G上为1％除外）。",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=810992122&format=interactive",
+  sheets_gid="306222260",
+  sql_file="web_vitals_by_ect.sql"
+  )
+}}
+
+TTFB受网络延迟和连接类型的影响。由上图可知，延迟越高、连接越慢，TTFB度量值就会越差。即使在被认为快速（4G）的移动端连接上，也只有21%的网站有一个快速的 TTFB。低于4G速度的站点基本上不会被评定为快速。
+
+我们来看看[2018年12月-2019年11月全球移动端网速](https://www.speedtest.net/insights/blog/content/images/2020/02/Ookla_Mobile-Speeds-Poster_2020.png)，可以看出整体上移动端连接网速并不高。那些网速和代表蜂窝网络的技术（比如5G）分布不均，并且最终会影响 TTFB。举个例子，[我们看尼日利亚的网络分布图]，该国大部分区域覆盖的都是2G和3G，极少数区域才有4G网络分布。
+
+令人惊讶的是，离线站点和通过4G网络访问的站点有着相同数目的良好的TTFB。通过 service workers，我们可以预期一些 TTFB 问题会得到缓解，但该趋势并未在上图中反映出来。
+
+## 性能监测的使用
+
+有许多不同的以用户为中心的指标可用于评估网站和应用程序。然而，有时预定义的指标并不完全符合我们的特定场景和需要。[性能监测 API](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceObserver)允许我们通过[自定义时间测量 API](https://developer.mozilla.org/en-US/docs/Web/API/User_Timing_API)、[长任务 API](https://developer.mozilla.org/en-US/docs/Web/API/Long_Tasks_API)、[事件计时 API](https://web.dev/custom-metrics/#event-timing-api)和[一些底层 APIs](https://web.dev/custom-metrics/)去获取自定义度量数据。举例来说，在这些API的帮助下我们可以记录页面转场时间或者量化服务端渲染应用的客户端激活。
```

Comment:
Can we check which are available as translations? Looks like first one is for example (didn't check them all): https://developer.mozilla.org/zh-CN/docs/Web/API/PerformanceObserver

Reply:
@bazzadp  do you mean I can just use MDN translation, if MDN has no corresponding translation, I can keep this original english name ?

Reply:
Correct. Nice for our readers to use the translation if there is one, and MDN is pretty good at this (as is Wikipedia), but yes sometimes there isn't one so have to stick with English version then.

Reply:
done

Reply:
I still only see the English links?

Reply:
sorry, my mistake, this time should be fixed